runtime/smithy-client/common/src/aws/smithy/kotlin/runtime/client/config/ClientSettings.kt

```diff
@@ -17,6 +18,12 @@ public object ClientSettings {
      */
     public val MaxAttempts: EnvironmentSetting<Int> = intEnvSetting("sdk.maxAttempts", "SDK_MAX_ATTEMPTS")
 
+    /**
+     * Specifies the minimum acceptable version of TLS to use when connecting to service endpoints.
+     */
+    public val MinTlsVersion: EnvironmentSetting<TlsVersion> =
+        enumEnvSetting<TlsVersion>("SDK_MIN_TLS", "sdk.minTls").orElse(TlsVersion.Tls1_2)
```

Comment:
**concern**: I think we really ought to use a namespace we own like `aws.smithy` or `aws.sdk` (I was late to other PR but left same comment for retries).

Reply:
I'm going to defer (re)naming this setting until standards exist for cross-SDK envvar/sysprop config naming conventions.